package.json

```diff
@@ -40,6 +43,9 @@
       "git add"
     ]
   },
+  "resolutions": {
+    "graphql": "^14.1.1"
+  },
```

Comment:
Apollo needs to be sure there's exactly one instance of `graphql` being used in the tree, so that version mismatches don't occur.